index.html

```diff
@@ -43,6 +43,122 @@ <h4>LET'S CONNECT</h4>
         </div>
     </div>
    </section>
+   <section id="portfolio">
+   <div class="container1">
+   <div class="card one">
+    <div class="placeholder-img1"></div>
+    <div class="text">
+        <h3>Tonic</h3>
+        <ul class="tech-dev">
+            <li><a href="#">CANOPY</a></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><a href="#">2015</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang-names">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+        </ul>
+        <button type="button">See Project</button>
+    </div>
```

Comment:
-  [ OPTIONAL ] As I can see the background color in your case is white but it is not white in Figma design so kindly make it the same color to match the Figma design as much as possible 💪🏻😊🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/200140957-97fe55d5-9013-4ad6-9af5-4fcd52379a5f.png)


- Figma design
![image](https://user-images.githubusercontent.com/58553711/200140930-2b25f23a-ef2e-4394-a6f6-2692366a1ff5.png)